README.md

```diff
@@ -35,28 +35,28 @@ Other documentation can be found in standalone READMEs:
 
 ## Getting started
 
-First, run the following to create a local environment file:
+Create an environment configuration file by running:
 
 ```bash
-cp .env.test .env.development.local
+touch .env
 ```
 
-and fill in any values, if required, by asking another member of the team.
+Now populate the file with secret manager services and the Oak config location. See [here](https://www.notion.so/oaknationalacademy/How-to-set-up-OWA-21ae880e58bb40bfaaa289030e984abd?pvs=4) for further details.
```

Comment:
This repo is open source, so we should probably remove the reference to an internal system. That's why the original instruction was a bit vague, although we could certainly improve on it.

Reply:
Understood: I've replaced the line with.
"Now populate the file with secret manager services and the Oak config location. Fill in other values, if required, by asking another member of the team."

Reply:
thanks @kimon-satan !